ros_buildfarm/config/release_build_file.py

```diff
@@ -108,6 +108,16 @@ def __init__(self, name, data):  # noqa: D107
         if 'upload_destination_credential_id' in data:
             self.upload_destination_credential_id = data['upload_destination_credential_id']
 
+        self.include_test_dependencies = True
+        self.run_package_tests = True
+        if 'package_dependency_behavior' in data:
+            if 'include_test_dependencies' in data['package_dependency_behavior']:
```

Comment:
Do you think we need to check that `package_dependency_behavior` is a dict before using the `in` keyword or is it acceptable to rely on the exception for configuration feedback?

Reply:
More checks more better.